chains/evm/contracts/onRamp/CCVProxy.sol

```diff
@@ -105,7 +107,7 @@ contract CCVProxy is IEVM2AnyOnRampClient, ITypeAndVersion, Ownable2StepMsgSende
   DynamicConfig private s_dynamicConfig;
 
   /// @dev The destination chain specific configs.
-  mapping(uint64 destChainSelector => DestChainConfig destChainConfig) private s_destChainConfigs;
+  mapping(uint64 destChainSelector => DestChainConfig destChainConfig) public s_destChainConfigs;
```

Comment:
This can never be public, did you mean to change this? Making our `s_` syntax names public is very ugly as it creates getters with the same `s_` name